File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int currValue; 
        for(int i = 0; i < nums.size(); i++){
            currValue = nums[i];
            for(int j = 0; j < nums.size(); j++){
                if(currValue == nums[j] && i != j){
                    return true;
                }

            }

        }
        return false;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> hash = new HashMap<>(); 

        for(int i = 0; i < nums.length; i++){
            //other number that i need 
            int otherIdxNum = target - nums[i];
            if(hash.containsKey(otherIdxNum)){
                return new int[] {hash.get(otherIdxNum), i};
            }
            
            hash.put(nums[i], i);
        }
        return new int[] {};
    }
}
